packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -61,20 +85,21 @@ const MAX_ATTESTATIONS_PER_GROUP = 3;
  * Note that we want to remove attestations with attesters that were included in the chain.
  */
 export class AggregatedAttestationPool {
-  private readonly attestationGroupByDataHashByIndexBySlot = new MapDef<
+  // per electra, different committees could have the same AttData hex
+  private readonly attestationGroupByIndexByDataHashBySlot = new MapDef<
```

Comment:
By swapping `ByIndex` and `ByDataHash`, it should take up less space. Wonder how much difference it would make. Too early to tell though

Reply:
we don't have other options due to the attestation consolidation requirement
I only have a concern for pre-electra, will add a TODO to verify it through benchmark